src/vl-body.js

```diff
@@ -0,0 +1,17 @@
+import { NativeVlElement, define } from '/node_modules/vl-ui-core/dist/vl-core.js';
+
+/**
+ * VlBody
+ * @class
+ * @classdesc 
+ * 
+ * @extends NativeVlElement
+ * 
+ * @see {@link https://www.github.com/milieuinfo/webcomponent-vl-ui-body/releases/latest|Release notes}
+ * @see {@link https://www.github.com/milieuinfo/webcomponent-vl-ui-body/issues|Issues}
+ * @see {@link https://webcomponenten.omgeving.vlaanderen.be/demo/vl-body.html|Demo}
+ * 
+ */
+export class VlBody extends NativeVlElement(HTMLBodyElement) {}
```

Comment:
moet ge nu nog nergens de stijl laden?

Reply:
Ja, het is een native element. Een niet native variant was ook een optie, maar de slot elementen krijgen stijl waar ze gedeclareerd worden. De enige optie dan was al de slot elementen opnieuw declareren door te clonen, maar dat heeft dan weer andere negatieve gevolgen.

Reply:
ja ergens hier in nen test (k weet nie meer waar) zegt ge wel "er wordt een minimale stijl geladen", maar das dan niet waar?

Reply:
heeft de component dan nog nut?

Reply:
Ja dat snap ik ook niet goed?

Reply:
Nee, voorlopig doet de body webcomponent nog niet veel. Hij fungeert momenteel alleen als reminder om de native core stijl toe te voegen, dus nu verpakt zit in de body zijn stijl wat iets logischer is dan een generieke core stijl te moeten inladen.